src/card.scss

```diff
@@ -0,0 +1,87 @@
+$hover-color: #00acdc;
+$hover-background-color: #fff;
+$hover-border-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  &__img {
+    display: block;
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__title {
+    margin: 40px 0 4px;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+  &__code {
+    margin-bottom: 16px;
+    font-weight: normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-main-color;
+  }
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+  &__stars {
+    display: flex;
+    align-content: center;
+  }
+  &__reviews {
+    font-weight: normal;
+    font-size: 10px;
+    color: $main-text-color;
+    text-align: center;
+  }
+  &__bottom {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    &_text {
+      font-weight: normal;
+      font-size: 12px;
+      line-height: 18px;
+      color: $second-main-color;
+    }
+    &_price {
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+  &__button {
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding